### PR TITLE
Pacman Easy

### PR DESCRIPTION
**Pull request by Prachi-05**

### CODE REVIEW

**Review by girlscript-blr — CHANGES_REQUESTED** [review]
Could you please upload these 2 .jpg files inside ur folder i.e. `Prachi-05:Pacman-Easy` and also modify the redirect path links for the images in your README.md file

You are not supposed to check-in anything outside your folder

Screenshot 2020-08-29 at 9 57 30 PM

File: Easy/2. Pacman Game/solutions/Prachi-05_Prachi/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Pacman Game - Easy Solution
+
+### by Prachi Sabarad
+
+For this easy level of Pacman, I've used the *pynput* module of **Python** to listen to keyboard key presses. 
+The maze is a 2-D array.
+Everytime it detects the Arrow keys or the WASD keys, the position of the Pacman, i.e. character '@' is altered in the maze and displayed on the screen.
+
+- [Initial position of the Pacman](/assets/images/pacmanGameEasy.jpeg) Initial position of the Pacman
+- [After several moves](/assets/images/pacmanMoving.jpg) Pacman after several moves
```
Comment (girlscript-blr) [inline Easy/2. Pacman Game/solutions/Prachi-05_Prachi/README.md]: Modify the image path links to the new path, after uploading the images inside your folder.

**Comment by Prachi-05** [comment]
Made the changes

**Review by girlscript-blr — CHANGES_REQUESTED** [review]

**Review by girlscript-blr — APPROVED** [review]

**Comment by girlscript-blr** [comment]
Greetings from Code With Girlscript Bangalore😃

Dear participant, we have gone through your pull request made in `EASY` sections of the problem statement.

Congratulations🎊 🎉  on getting your pull request successfully merged in the `Pacman Game - Easy` section in `code-with-girlscript-bangalore` repository. We will be distributing your valuable badges🥇🥈🥉🎖️ corresponding to the pull requests that are merged.

Learning is a never-ending process. 📈 Check out the [Intermediate](https://github.com/girlscript-blr/code-with-girlscript-bangalore/tree/solution/Medium/2.%20Pacman%20Game) & [Advanced](https://github.com/girlscript-blr/code-with-girlscript-bangalore/tree/solution/Hard/2.%20Pacman%20Game) level problem statements and try your hand at it for earning higher-level badges. Thanks for your contribution! Happy coding😃

**Pull request merged**